tests/test_sse.py

```diff
@@ -535,19 +464,18 @@ async def func(request: web.Request) -> web.StreamResponse:
         app = web.Application()
         app.router.add_route("GET", "/", func)
 
-        host = "127.0.0.1"
-        runner = await make_runner(app, host, unused_tcp_port)
-        url = f"http://{host}:{unused_tcp_port}/"
+        client = await aiohttp_client(app)
+        async with client.get("/") as resp:
+            assert resp.status == 200
 
         last_event_id = "42"
         headers = {EventSourceResponse.DEFAULT_LAST_EVENT_HEADER: last_event_id}
-        resp = await session.request("GET", url, headers=headers)
-        assert resp.status == 200
+        async with client.get("/", headers=headers) as resp:
```

Comment:
Curious that you put the async with here, as this is the one place where it actually shouldn't matter (although I'd still recommend using it).

I'll just merge it like this, but if warnings appear in future, the other tests may need the context manager too.